styleguide/components/31-molecules/status-message/_status-message.scss

```diff
@@ -37,10 +33,18 @@
     text-align: center;
   }
 
+  p {
```

Comment:
Logischer is eigenlijk om dit vanaf lijn 70 te laten starten, zo staan de concats eerst ;-) En daarna pas de onderliggende elementen :D

Reply:
Fixed.